Updates/1559_Saltgurka-Molten_Core_Fixes.sql

```diff
@@ -0,0 +1,386 @@
+-- Thanks to malcrom for providing movement data: https://github.com/cmangos/issues/issues/892#issuecomment-270154679  /  http://pastebin.com/U87kY1UR
+
+-- ************************
+-- MISC
+-- ************************
+-- Add the circle fire animation to Rune of Zeth
+DELETE FROM `gameobject` WHERE `guid`=113269;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES 
+('113269', '178188', '409', '748.839', '-985.16', '-178.283', '3.14159', '0', '0', '-1', '0', '604800', '255', '1');
+
+-- Set animprogress to 100 for all runes. The fire spout animation doesn't play otherwise.
+UPDATE `gameobject` SET `animprogress`=100 WHERE `id` IN(176951,176952,176953,176954,176955,176956,176957);
+
+-- Sulfuron Harbinger followers should not respawn while boss is dead
+UPDATE `creature_linking_template` SET `flag`=1031 WHERE `master_entry`=12098 AND `entry`=11662; -- Old flag: 7
+
+-- Gehennas followers should not respawn while boss is dead
+UPDATE `creature_linking_template` SET `flag`=1031 WHERE `master_entry`=12259 AND `entry`=11661; -- Old flag: 7
+
+-- ************************
+-- Garr movement/linking fixup
+-- ************************
+UPDATE `creature` SET `spawndist`=10,`MovementType`=1 WHERE `guid`=56609; -- Garr
+UPDATE `creature` SET `position_x`=684.192,`position_y`=-506.928,`position_z`=-214.424,`orientation`=3.73863 WHERE `guid`=56627; -- Firesworn
+
+-- Firesworn follows Garr. To be blizzlike they should remain in a perfect circle around him at all times, but that's not possible with the current linking system
+UPDATE `creature_linking_template` SET `flag`=1543 WHERE `entry`=12099 AND `master_entry`=12057; -- Old flag: 7
+
+-- ************************
+-- Core Hounds
+-- ************************
+-- Pathing for Ancient Core Hound Entry: 11673 'TDB FORMAT' 
+-- Original had duplicate points
+SET @GUID := 56853;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2 WHERE `guid`=@GUID;
+DELETE FROM `creature_movement` WHERE `id`=@GUID;
+INSERT INTO `creature_movement` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`) VALUES

```

Comment:
Yes @cala , there is something wrong here, but use

INSERT INTO `creature_movement` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`) VALUES
(@GUID,1,1051.314,-799.6896,-151.9713,100),
(@GUID,2,1039.667,-791.0968,-151.0096,100),
(@GUID,3,1026.315,-767.8003,-156.6493,100),
(@GUID,4,1038.774,-761.6977,-152.6580,100),
(@GUID,5,1064.627,-748.9409,-151.7516,100),
(@GUID,6,1038.774,-761.6977,-152.6580,100),
(@GUID,7,1026.315,-767.8003,-156.6493,100),
(@GUID,8,1039.641,-791.0518,-150.9892,100),
(@GUID,9,1051.314,-799.6896,-151.9713,100),
(@GUID,10,1040.764,-815.1115,-152.1656,100);

instead.